src/authhandler.cpp

```diff
@@ -66,23 +66,23 @@ AuthHandler::AuthHandler(Accounts::Manager *manager, const quint32 accountId, co
 
 bool AuthHandler::init()
 {
-    FUNCTION_CALL_TRACE;
+    FUNCTION_CALL_TRACE(lcCalDav);
```

Comment:
In the corresponding Buteo PR, the `FUNCTION_CALL_TRACE` macro is used with the trace variant of the logging category. Why not in CalDAV also ?

Reply:
Yes this should also have a trace variant, thanks for picking that up. Fixed now.